### PR TITLE
Fix/documentation

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: manager/manager.py [hunk 1]
```diff
@@ -1,126 +1,159 @@
...
+
+
+class BusManager:
+    """
+    The manager of the bus.
+    Puts data on the bus and returns it from the bus.
+    """
+    def __init__(self):
+        self.processing_lock = Lock()
+        """ """
```
Comment (LRstudentHU) [inline manager/manager.py]: This is empty

File: manager/manager.py [hunk 2]
```diff
@@ -1,126 +1,159 @@
...
+        self.should_stop = False
+        """Contains if the bus must be ended."""
```
Comment (LRstudentHU) [inline manager/manager.py]: Maybe add some newlines between members; it is confusing whether the top or bottom comment belongs to the variable now

File: manager/manager.py [hunk 3]
```diff
@@ -1,126 +1,159 @@
...
+        for frame in to_send:
+            # Distribute frame internally
+            self.rx_queue.append(frame)
+
+            #print(frame)  # 'send'
+            print()
+
+    def _process_rx(self):
+        """
+        Function processes an incomming frame.
```
Comment (LRstudentHU) [inline manager/manager.py]: Spelling

File: modules/button_module/main.py [hunk 4]
```diff
@@ -27,6 +41,9 @@ def main():
 
 
 def stop(signal, frame):
+    """
+    Stops the process and  stops the listening to incomming frames
```
Comment (LRstudentHU) [inline modules/button_module/main.py]: Spelling

File: modules/controller_module/main.py [hunk 5]
```diff
@@ -21,6 +21,9 @@ def main():
 
 
 def stop(signal, frame):
+    """
```
Comment (LRstudentHU) [inline modules/controller_module/main.py]: Empty comment

**Review by LRstudentHU — CHANGES_REQUESTED** [review]

File: manager/manager.py (hunk 1 above)
Comment (sebastiaan1997) [reply manager/manager.py]: Working on it

**Comment by steviestickman** [comment]
solves #17, #18  and #22

**Review by LRstudentHU — APPROVED** [review]

**Pull request merged**